packages/ripple-nuxt-tide/modules/data-driven-component/components/TideMapPreventionFamilyViolence/TideMapCard.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="tide-map-card" v-on:click="cardClicked">
```

Comment:
We use BEM as css naming conventions in this project. http://getbem.com/naming/
The class here should be `tide-map-card`, `tide-map-card__header`, `tide-map__marker` and `tide-map-card__arrow`. Could you please change it to our standard? Should include all css in this PR.